src/main/java/org/threeten/extra/scale/TaiInstant.java

```diff
@@ -141,7 +141,7 @@ public static TaiInstant ofTaiSeconds(long taiSeconds, long nanoAdjustment) {
     }
 
     /**
-     * Obtains an instance of {@code TaiInstant} from an {@code Instant}.
+     * Obtains an instance of {@code TaiInstant} from an {@link Instant}.
```

Comment:
I follow a coding rule whereby the first sentence of javadoc does not include `@link`. This avoids links being visible when the first sentence is used as a summary.

Reply:
OK. Have switched the link to the first non-summary usage.